### ISSUE TITLE
Error trying to build

### ISSUE THREAD

**Issue by GuilhermeRaposo** [pettiboy/react-ui-scrollspy#9]
I get this error after running npm install and npm run build
```
react-ui-scrollspy@2.1.0 build
rollup -c

Error loading `tslib` helper library.
[!] Error: Package subpath './package.json' is not defined by "exports" in D:\Side Projects\react-ui-scrollspy\node_modules\tslib\package.json
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './package.json' is not defined by "exports" in D:\Side Projects\react-ui-scrollspy\node_modules\tslib\package.json
    at new NodeError (node:internal/errors:393:5)
    at exportsNotFound (node:internal/modules/esm/resolve:295:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:631:9)
    at resolveExports (node:internal/modules/cjs/loader:538:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:607:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1025:27)
    at Function.Module._load (node:internal/modules/cjs/loader:885:27)
    at Module.require (node:internal/modules/cjs/loader:1105:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (D:\Side Projects\react-ui-scrollspy\node_modules\rollup-plugin-typescript2\src\tslib.ts:11:23)
```

**Comment by GuilhermeRaposo** [comment]
Fixed it by downgrading Node to 16.18.1

**Comment by pettiboy** [comment]
updating `rollup-plugin-typescript2` to `0.31.0` fixes this. checkout #11.
